### ISSUE TITLE
dm_rockso not 3.6 compatable

### ISSUE THREAD

**Issue by nopthetop** [Coalfire-Research/DeathMetal#1]
The call to asyncio.run() on line 71 of dm_rockso is not compatible with Python 3.6.  Causes the program to throw an exception and fail. Python 3.7 seems to work okay, had to upgrade.  Not sure if it's a bug, but figured you'd want to know for compatibility reasons.

**Comment by panch0** [comment]
You are correct, I gotta either make it explicitly clear that 3.7 is required or better yet make the implementation support 3.6. I'll take a look sometime tonight.

Also, to be honest rockso needs to be re-implemented. The way its written right now it basically spams packets and waits for 5 seconds for responses to come in. Won't scale well to larger subnets. I think the right way to go about it is to wrap each request in a task object with a 5 second lifetime. I should have time tonight or this weekend to fix it!

Thanks for the heads up,
Victor

**Comment by biorchang** [comment]
I experienced the same error using Python 3.7.  I am using a clean W10 build with the latest Python install and ran setup.py to install dependencies.  Python in PATH.
